src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -35,6 +35,10 @@
 * Needs to be "without sharing" so that the Install Script can run it.
 */
 public without sharing class UTIL_CustomSettingsFacade {
+    
+   	public static boolean userCanEditDataImportSettings {get {
```

Comment:
I pulled it into the facade without a full knowledge of how the facade wants to be structured. I don't know if there's a world where the facade cares about user object permissions.

I think it'd be safe just to be in the controller.